internal/telemetry/client.go

```diff
@@ -313,9 +356,7 @@ func (c *Client) flush() {
 		c.newMetrics = false
 		r := c.newRequest(RequestTypeGenerateMetrics)
 		payload := &Metrics{
-			Namespace:   c.Namespace,
-			LibLanguage: "go",
-			LibVersion:  version.Tag,
```

Comment:
Why delete these two?

Reply:
Lib fields were removed from metrics for v2 and moved to the headers [here](https://github.com/DataDog/dd-trace-go/blob/19f0596599a3003c655c887624eac2c57d7956f3/internal/telemetry/client.go#LL447-L448C47)